base.lua

```diff
@@ -152,7 +152,7 @@ function Qless.jobs(now, state, ...)
     elseif state == 'depends' then
       return queue.depends.peek(now, offset, count)
     elseif state == 'recurring' then
-      return queue.recurring.peek(math.huge, offset, count)
+      return queue.recurring.peek('+inf', offset, count)
```

Comment:
Sorry. Miss this messages somehow.
It is not defined how convert `math.huge` to string representation.
So it totally platfrom specific e.g. [Serialization of math.huge](http://lua-users.org/lists/lua-l/2012-05/msg00653.html)
But redis expect only `[+-]inf`.